### PR TITLE
Remove exclusions for readme

### PR DESCRIPTION
**Pull request by AlexGhiondea**

### CODE REVIEW

**Review by chidozieononiwu — APPROVED** [review]

File: eng/.docsettings.yml [hunk 1]
```diff
@@ -121,21 +121,14 @@ known_presence_issues:
 known_content_issues:
   - ['README.md','Root readme']
   - ['doc/README.md','Doc readme']
-  - ['sdk/core/Microsoft.Extensions.Azure/README.md','#5499']
   - ['sdk/keyvault/Microsoft.Azure.KeyVault/README.md','#5499']
   - ['sdk/servicebus/Microsoft.Azure.ServiceBus/README.md','#5499']
   - ['sdk/eventhub/Microsoft.Azure.EventHubs/README.md','#5499']
-  - ['sdk/eventhub/Azure.Messaging.EventHubs.Shared/README.md','#5499']
```
Comment (jsquire) [inline eng/.docsettings.yml]: This is the shared code project for Event Hubs, containing only internal types that are used by multiple Event Hubs packages.   It should not be validated as a product ReadMe.

Please consider leaving this exclusion.

**Review by jsquire — CHANGES_REQUESTED** [review]

**Comment by maririos** [comment]
TA has been fixed in separate PRs

File: eng/.docsettings.yml (hunk 1 above)
Comment (AlexGhiondea) [reply eng/.docsettings.yml]: I have added this back

Comment (jsquire) [reply eng/.docsettings.yml]: Thanks!

**Review by jsquire — APPROVED** [review]

**Comment by AlexGhiondea** [comment]
Closing this PR as the 2 libraries that needed to be fixed (core/textanalytics) have been resolved and the exclusions removed.

**Pull request closed without merging**